platform/viewer/cypress/integration/common/OHIFStudyList.spec.js

```diff
@@ -147,7 +148,8 @@ describe('OHIF Study List', function() {
       //Wait result list to be displayed
       cy.waitStudyList();
       cy.get('@searchResult').should($list => {
-        expect($list.length).to.be.eq(16); // TODO: Where are you hiding MISTER^MR?
+        // TODO: Why are we facing some inconsistency with this result? ¯\_(ツ)_/¯
```

Comment:
Yeah we generally need to improve the e2e tests sometime, pointing at a public repo that has STOW functionality in the app makes these kinds of tests super brittle.